tests/FSharp.Compiler.ComponentTests/EmittedIL/SeqExpressionStepping/SeqExpressionStepping.fs

```diff
@@ -14,47 +14,104 @@ module SeqExpressionStepping =
         |> withNoOptimize
         |> withEmbeddedPdb
         |> withEmbedAllSource
+        |> withRealInternalSignatureOn
```

Comment:
Some tests in this file have `withRealInternalSignatureOff`, but they still then call `verifyCompilation`, which will add `--realsig+` back, no? Did you mean to remove this from here? @KevinRansom

Reply:
...And wouldn't the alternating tests overwrite each other's baseline files anyway, since there are no separate baseline suffixes specified?

Reply:
> ...And wouldn't the alternating tests overwrite each other's baseline files anyway, since there are no separate baseline suffixes specified?

@brianrourkeboll 
Let me take a look.  TBH --- I am now hating all of the IL baseline tests.  Just on a general, "I hate them" basis.

Reply:
@brianrourkeboll 
You are correct I will fix the tests,
thanks
Kevin

Reply:
@KevinRansom Or I could in my PR (#16650), if that would be easier? Otherwise I'll just have more merge conflicts to resolve lol

Reply:
I got it, here she blows matey:  https://github.com/dotnet/fsharp/pull/16795

Reply:
@brianrourkeboll , please look it over and I will get it merged.